birdhouse/config/catalog/config/magpie/providers.cfg.template

```diff
@@ -8,7 +8,7 @@ providers:
     sync_type: wps
 
   ${CATALOG_THREDDS_SERVICE}:
-    url: http://${PAVICS_FQDN}:8083/twitcher/ows/proxy/thredds
+    url: http://thredds:8080/twitcher/ows/proxy/thredds
```

Comment:
Will need to check with @tlvu. 

There was an issue regarding THREDDS that needed the full `${PAVICS_FQDN_PUBLIC}${TWITCHER_PROTECTED_PATH}` to be defined as Magpie's URL so that Twitcher could redirect in the right location. Not sure if this impacts the catalog browsing/listing/access or not.

Reply:
I actually didn't even mean to make this change since catalog is being deprecated. It must have slipped in from the cherry-pick from the other branch.

I'm ok to either revert this change or leave it as is (since catalog is deprecated)

Reply:
If it works with `thredds:8080`, then it is better to change it. 
It is guaranteed to fail with `${PAVICS_FQDN}:8083` after changes are applied.